site/_docs/index.md

```diff
@@ -39,8 +39,17 @@ point it at some data.
 
 {% highlight java %}
 public static class HrSchema {
-  public final Employee[] emps = 0;
-  public final Department[] depts = 0;
+public final Employee[] emps = {
```

Comment:
or
```
  public final Employee[] emps = new Employee[0];
  public final Department[] depts = new Department[0];
```

Reply:
This is supposed to be a schema, not a database, so I expect it should contain no data.

Reply:
done

Reply:
Is there a test which uses this code?
This would increase our confidence that the example is good.
In the past I have discovered that documentation is wrong if there are no tests validating it.

Reply:
@mihaibudiu It doesn't look like there is one at the moment, do you want me to add one?

Only a series of tests based on [HrSchema](https://github.com/apache/calcite/blob/main/testkit/src/main/java/org/apache/calcite/test/schemata/hr/HrSchema.java)

Reply:
I totally agree with you, the calcite documentation is very rudimentary.

Reply:
having a test is best. But if you have written a test (not part of the PR) based on this structure which works as expected, it's good too.

Reply:
```
public class Main {
  public static class HrSchema {
    public final Employee[] emps = new Employee[0];
    public final Department[] depts = new Department[0];
  }

  public static void main(String[] args) {
    try {
      Class.forName("org.apache.calcite.jdbc.Driver");
      Properties info = new Properties();
      info.setProperty("lex", "JAVA");
      Connection connection = DriverManager.getConnection("jdbc:calcite:", info);
      CalciteConnection calciteConnection = connection.unwrap(CalciteConnection.class);
      SchemaPlus rootSchema = calciteConnection.getRootSchema();
      Schema schema = new ReflectiveSchema(new HrSchema());
      rootSchema.add("hr", schema);
      Statement statement = calciteConnection.createStatement();
      ResultSet resultSet = statement.executeQuery(
          "select d.deptno, min(e.empid)\n"
              + "from hr.emps as e\n"
              + "join hr.depts as d\n"
              + "  on e.deptno = d.deptno\n"
              + "group by d.deptno\n"
              + "having count(*) > 1");
      print(resultSet);
      resultSet.close();
      statement.close();
      connection.close();
    } catch (Exception e) {
      e.printStackTrace();
    }
  }
}
```
@mihaibudiu Local testing looks fine